synapse/handlers/federation.py

```diff
@@ -1816,7 +1818,14 @@ def prep(ev_info: _NewEventInfo):
         )
 
     @defer.inlineCallbacks
-    def _persist_auth_tree(self, origin, auth_events, state, event):
+    def _persist_auth_tree(
+        self,
+        origin: str,
+        auth_events: List[EventBase],
+        state: List[EventBase],
+        event: EventBase,
+        room_version: RoomVersion,
```

Comment:
a word or two about this in the docstring wouldn't hurt. Where does it come from?